src/utils/mediaQueries.ts

```diff
@@ -33,7 +41,10 @@ export const MEDIA_QUERY_NAMES = MEDIA_QUERY_MATCHES.map(({ name }) => name).con
 export const subscribeToScreenSizeChange = (callback: (event: MediaQueryListEvent) => void): Command => {
   const mediaQueryLists = MEDIA_QUERIES.map((query) => window.matchMedia(query))
 
-  mediaQueryLists.forEach((mql) => mql.addListener(callback))
+  mediaQueryLists.forEach((mql) => mql.addEventListener('change', callback))
 
-  return (): void => mediaQueryLists.forEach((mql) => mql.removeListener(callback))
+  return (): void => mediaQueryLists.forEach((mql) => mql.removeEventListener('change', callback))
 }
+
+// media queries below medium size
+export const MediumDownQueries = [TypeMediaQueries.MD, TypeMediaQueries.SM, TypeMediaQueries.XS]
```

Comment:
I'd rather remove this in favor of having something like I commented above

```js
const isDesktop = useMediaBreakpoint(['xl', 'lg'])
```

Reply:
In this case, wouldn't the **enum** be better to avoid having to type the types manually?

const isDesktop = useMediaBreakpoint([TypeMediaQueries.XL, TypeMediaQueries.LG])